android/src/main/java/com/pinmi/react/printer/adapter/PrinterAdapter.java

```diff
@@ -30,4 +30,5 @@
 
     public void connectAndSend(String host, Integer port, String rawBase64Data, Callback successCallback, Callback errorCallback);
 
+    public void connectAndSend(PrinterDeviceId printerDeviceId, String data, Callback successCallback, Callback errorCallback);
```

Comment:
what is PrinterDeviceId? Its a type ?

Reply:
It is printer device object which holds printer info